Main.java

```diff
@@ -9,7 +11,22 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+
+    //Enter line that asks the user to enter an interger
+    System.out.println("Please enter an interger:");
+    int a = input.nextInt();
+
+    //Enter line that asks the user to enter another interger
+    System.out.println("Please enter another interger:");
+    int b = input.nextInt();
+
+    //Sorts the numbers in acending order
+    if (a > b) {
+    System.out.println("Your numbers in ascending order are " + b + ", " + a); 
+    } else if (a < b) {
+    System.out.println("Your numbers in ascending order are " + a + ", " + b);
+    }

```

Comment:
Watch your indentation. If there is code inside of an if statement, it should be TABBED over an extra bit to show it is inside of the if-statement.